src/libs/antares/study/parts/hydro/container.cpp

```diff
@@ -355,8 +355,8 @@ bool PartHydro::checkReservoirLevels(const Study& study)
         for (unsigned int day = 0; day < DAYS_PER_YEAR; day++)
         {
             if (!errorLevels
-                && (colMin[day] < 0 || colAvg[day] < 0 || colMin[day] > colMax[day]
-                    || colAvg[day] > 100 || colMax[day] > 100))
+                && (0 > colMin[day] || colMin[day] > colAvg[day] || colAvg[day] > colMax[day]
+                    || colMax[day] > 100))
```

Comment:
@flomnes Also, variables colMin[day], colMax[day], colAvg[day] contains values between 0-1, so comparing values with 100 is a bug, we need to compare it with 1.
Would you be able to confirm that?